### PR TITLE
Fixed-Issue-#1285 Inconsistent character limit during organization cr…

### LINKED ISSUE #1285: Inconsistent character limit during organization creation for both name and description.
**Is your feature request related to a problem? Please describe.**
When creating an organization, there is currently no enforced character limit for the organization's name and description. This lack of limitation allows users to input excessively long strings, which may lead to issues related to clarity and system performance. 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Approach to be followed (optional)**
A clear and concise description of approach to be followed.

**Additional context**
image

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by gauravsingh94**
…eation for both name and description.

**What kind of change does this PR introduce?**

**Issue Number:**

Fixes #1285 

**Did you add tests for your changes?**

**Snapshots/Videos:**

https://github.com/PalisadoesFoundation/talawa-admin/assets/99260988/ea0edc92-67d5-40c9-955f-190506e62e1b

**If relevant, did you update the documentation?**

**Summary**

Now we can enter only 50 character for the organization name and 200 for the organization description and location.

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by gauravsingh94** [comment]
I have Updated it

50 characters for the organization name.
200 characters for the organization description.
100 characters for the organization location.

File: src/screens/OrgList/OrgList.tsx [hunk 1]
```diff
@@ -478,10 +481,13 @@ function orgList(): JSX.Element {
                 required
                 value={formState.descrip}
                 onChange={(e): void => {
-                  setFormState({
-                    ...formState,
-                    descrip: e.target.value,
-                  });
+                  const descriptionText = e.target.value;
+                  if (descriptionText.length < 100) {
```
Comment (noman2002) [inline src/screens/OrgList/OrgList.tsx:485]: Change it to 200

**Review by noman2002 — CHANGES_REQUESTED** [review]

File: src/screens/OrgList/OrgList.tsx (hunk 1 above)
Comment (gauravsingh94) [reply src/screens/OrgList/OrgList.tsx]: Done 👍.

**Review by aashimawadhwa — APPROVED** [review]
LGTM 👍🏼 @noman2002  can be merged

**Comment by rishav-jha-mech** [comment]
LGTM

**Review by rishav-jha-mech — APPROVED** [review]

**Review by noman2002 — APPROVED** [review]

**Pull request merged**